packages/fxa-auth-server/lib/geocoding.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  Client,
+  PlaceType2,
+  Status,
+} from '@googlemaps/google-maps-services-js';
+import countries from 'i18n-iso-countries';
+import { ConfigType } from '../config';
+import { Logger } from 'mozlog';
+const config: ConfigType = require('../config').getProperties();
+
+export type ZipLocation = {
+  city: string;
+  country: string;
+  countryCode: string;
+  state: string;
+  stateCode: string;
+};
+
+/**
+ *
+ * @param zip
+ * @param country - Two-letter country code ([ISO 3166-1 alpha-2] or corresponding Country name
+ * @returns
```

Comment:
> This seems incomplete here, and also since this is a TS file, it seems unnecessary, as we can specify the types of the parameters in the function declaration?

Agreed, I've been removing these from TS files I come across as I see them. VS Code has a doc snippet that sticks these in that's kind of annoying.

Reply:
Actually upon talking with Ben about this, it's better not to tie these inputs to Stripe types, as Stripe can change their API. Ideally we use our own types or the types for the libraries involved in this module. Sorry for the confusion.